### ISSUE TITLE
Evaluate the performance of a model

### ISSUE THREAD

**Issue by linchaozhang** [ildoonet/tf-pose-estimation#121]
how to Evaluate the performance of a model on coco, there is only the loss, not accuracy

**Comment by linchaozhang** [comment]
@ouceduxzk 
thanks for your reply, i have evaluated on coco dateset, but the AP is poor, about only 0.19 for AP0.5, how is your result?

**Comment by ouceduxzk** [comment]
can you share your evaluation script? currently, I still have one problem with my evaluation script, correct me if i am wrong, the coco  validation images can have multiple persons, but in the json label file, each person comes with a standalone keypoints label, which means you do not know how many person in one image is labeled.       Then in the prediction phase, our model predicts n person's keypoints, there might be some person too small to be predicted, which caused a mismatch of #of person in the pred.json vs the groudtruth, leading to the crash when calling cocoapi

**Comment by ildoonet** [comment]
https://github.com/ildoonet/tf-pose-estimation/blob/master/etcs/experiments.md

This is the latest results. This is the same result as the original openpose with single scale.

**Comment by AndrewSoul** [comment]
Can I ask that what's the difference between cmu_model and original Openpose? Did the net structure change?

**Comment by Yuliang-Zou** [comment]
Hi @ildoonet , I trained the 'vgg' model for ~135k iterations (batch size is 16), the model seems to converge according to the training curve. However, the AP on val2014 is only 0.35. Do you have any ideas what could be the reason?

**Comment by Yuliang-Zou** [comment]
Hi @ildoonet , did you use your eval code to get the results? If I used your pre-trained openpose model and evaluated using your code, the mAP is only ~0.43. 

And seems that the default setting in your eval code is wrong. You resize the input image into a fixed shape without maintaining the ratio of that image.

**Comment by HONGJUNRL** [comment]
> https://github.com/ildoonet/tf-pose-estimation/blob/master/etcs/experiments.md
> 
> This is the latest results. This is the same result as the original openpose with single scale.

how to evaluate with multi scale?

**Comment by HONGJUNRL** [comment]
@Yuliang-Zou ，hello, do you know how to evaluate with multi scale?

**Comment by Yuliang-Zou** [comment]
@HONGJUNRL You can check this repo: https://github.com/ZheC/Realtime_Multi-Person_Pose_Estimation

**Comment by niazahamd89** [comment]
Hi forks how i can generate the results for my model i can plot the pose on picture after training but how i can store the results and convert it to coco format to evaluate on coco metric.
